src/mixings/dependent_mixing.h

```diff
@@ -1,8 +1,6 @@
 #ifndef BAYESMIX_MIXINGS_DEPENDENT_MIXING_H_
 #define BAYESMIX_MIXINGS_DEPENDENT_MIXING_H_
 
-#include <Eigen/Dense>
-
 #include "base_mixing.h"
 
 class DependentMixing : public BaseMixing {
```

Comment:
Probably I missed this, but what is the purpose of `DependentMixing`?

Reply:
It's just preparation for the introduction of the probit mixing. It's part of the "light mixing refactoring" I was talking about.